README.md

```diff
@@ -1,7 +1,72 @@
-#Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Kailash Khadka . Jarvis Consulting
+
+A recent graduate from the University of Waterloo with a specialization in AI and Machine Learning. Was a teacher, had been an engineer, now an aspiring Software Engineer but always a student. A tech guy who loves to tinker with software and hardware equally. Passionate about developing smart solutions to problems. Looking to contribute to the growth of my employers while growing alongside.
```

Comment:
CIBC exp and tech skilss

Reply:
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer